specification/subscription/resource-manager/readme.md

```diff
@@ -34,6 +34,18 @@ openapi-type: arm
 tag: package-2019-10-preview
```

Comment:
Would you like to change the default tag to `package-2020-09` as well?

Reply:
No, we don't it right now, once we are ready to generate SDKs from this version, I will send out another PR with that change

Reply:
Changed it to new version